Casks/m/mullvadvpn.rb

```diff
@@ -2,14 +2,15 @@
   version "2025.2"
   sha256 "f5fdbe009489ef6c3f2d5182ca5b460dae555fbcbe3c132a9b7ff9316e41be0f"
 
-  url "https://cdn.mullvad.net/app/desktop/releases/#{version}/MullvadVPN-#{version}.pkg"
+  url "https://github.com/mullvad/mullvadvpn-app/releases/download/#{version}/MullvadVPN-#{version}.pkg",
+      verified: "github.com/mullvad/mullvadvpn-app/"
   name "Mullvad VPN"
   desc "VPN client"
   homepage "https://mullvad.net/"
 
   livecheck do
-    url "https://mullvad.net/download/app/pkg/latest/"
-    strategy :header_match
+    url :url
+    strategy :github_latest
```

Comment:
```suggestion
    url "https://mullvad.net/download/vpn/macos"
    regex(/href=.*?MullvadVPN[._-]v?(\d+(?:\.\d+)+)\.pkg/i)
```

Upstream also creates GitHub releases for Android and iOS, so the "latest" release isn't guaranteed to be for the desktop app. Without other options, we would have to use `GithubReleases` to check multiple releases to identify the newest desktop version. Luckily, the [first-party download page for macOS](https://mullvad.net/download/vpn/macos) also links to the newest pkg file from GitHub, so we can check that (i.e., `GithubReleases` responses can be fairly large, so we try to avoid it when we can).

Reply:
Thank you for the suggestion. I've committed it, let's see how the CI responds to it